float32 motor1
float32 motor2
